pkg/models/datasource_cache.go

```diff
@@ -319,3 +321,14 @@ func ClearDSDecryptionCache() {
 
 	dsDecryptionCache.cache = make(map[int64]cachedDecryptedJSON)
 }
+
+func awsServiceNamespace(dsType string) string {
+	switch dsType {
+	case DS_ES, DS_ES_OPEN_DISTRO:
+		return "es"
+	case DS_PROMETHEUS:
+		return "aps"
+	default:
+		panic(fmt.Sprintf("Unsupported datasource %s", dsType))
```

Comment:
I would quote `dsType`, in case it contains whitespace:

```suggestion
		panic(fmt.Sprintf("Unsupported datasource %q", dsType))
```